docs/release-notes/dont-send-docker-creds-to-any-server.txt

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+**Improvements**
+
+-  Security: Setting ``registry_auth.serveraddress`` will now only send credentials to the server
+   configured. Not setting ``registry_auth.serveraddress`` is now deprecated and will be required
+   soon.
```

Comment:
(failed to click "add comment" on this earlier, sorry)

Probably worth mentioning that this only applies when `registry_auth` is set.  It's totally fine to not set `registry_auth.serveraddress`, so long as `registry_auth == none`